docker-compose.dep.yml

```diff
@@ -81,32 +81,15 @@ services:
             - POSTGRES_DB=wallets
             - POSTGRES_USER=postgres
             - POSTGRES_PASSWORD=dbpass
-                
-    tails-server:
-        image: kivaprotocol/tails-server:latest
-        container_name: tails-server
-        ports:
-            - 6543:6543
-        networks:
-            - agency-network
-        depends_on:
-            - protocol-indy-pool
-        command: >
-            tails-server
-                --host 0.0.0.0
-                --port 6543
-                --storage-path /tmp/tails-files
-                --log-level INFO
 
     multitenant:
         image: bcgovimages/aries-cloudagent:py36-1.15-1_0.6.0
-        container_name: multitenant
+        container_name: ac-multitenant
         networks:
           - agency-network
         depends_on:
           - protocol-indy-pool
```

Comment:
do you need to update the dependency containers

Reply:
The good thing is the networking name stays the same so all the connections still work, it's just the saved container name that changes